publish-wrims-template/build.gradle

```diff
@@ -0,0 +1,102 @@
+plugins {
+    id 'maven-publish'
+    id 'distribution'
+}
+
+def zipFilePath = project.findProperty('wrimsGuiZipFile') ?: 'C:/WRIMS/wrims_gui_template.zip'
```

Comment:
I think template file can be store at github. It may need change if we delete some files from the template file. Otherwise, it generally would not change. To add some files to the resource file, we can update the resources file on github

Reply:
Josh updated to pull from open data